### ISSUE TITLE
Unescaped left brace in regex is illegal here in regex

### ISSUE THREAD

**Issue by eserte** [higemaru/p5-text-hikidoc#1]
Compilation of the module fails with bleadperl (e.g. perl 5.25.4):

```
#   Failed test 'use Text::HikiDoc;'
#   at t/00_lib-Text-HikiDoc_compile.t line 5.
#     Tried to use 'Text::HikiDoc'.
#     Error:  Unescaped left brace in regex is illegal here in regex; marked by <-- HERE in m/{{ <-- HERE ([^\s\(\)\'\"]+)([\000-\377]*)}}/ at /tmpfs/.cpan-build-cpansand/2016082818/Text-HikiDoc-1.021-0/blib/lib/Text/HikiDoc.pm line 705.
# Compilation failed in require at t/00_lib-Text-HikiDoc_compile.t line 5.
# BEGIN failed--compilation aborted at t/00_lib-Text-HikiDoc_compile.t line 5.
# Looks like you planned 3 tests but ran 1.
# Looks like you failed 1 test of 1 run.
t/00_lib-Text-HikiDoc_compile.t ................... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 3/3 subtests 
...
```

**Comment by higemaru** [comment]
thx for your report. I just fixed.

**Comment by eserte** [comment]
I still see failures with recent version 1.022:

```
Unescaped left brace in regex is illegal here in regex; marked by <-- HERE in m/({{ <-- HERE |}})/ at /tmpfs/.cpan-build-cpansand/2016102521/Text-HikiDoc-1.022-1/blib/lib/Text/HikiDoc.pm line 205, <DATA> line 1.
# Looks like your test exited with 255 before it could output anything.
t/20_lib-Text-HikiDoc_basic.t ..................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 92/92 subtests 
```

**Comment by higemaru** [comment]
Ooops. I fixed and tested on perl-5.25.6. Now recent version is 1.023.

**Comment by eserte** [comment]
Confirmed. No problems now with 5.25.6.
